activerecord/test/schema/schema.rb

```diff
@@ -1263,6 +1263,13 @@
     t.integer :id
     t.datetime :created_at
   end
+
+  create_table :test_exclusion_constraints, force: true do |t|
```

Comment:
this should be in https://github.com/rails/rails/blob/main/activerecord/test/schema/postgresql_specific_schema.rb

Reply:
Yes. This table is only required for postgresql adapter.